smurfs/src/components/App.js

```diff
@@ -1,22 +1,56 @@
 import React, { Component } from 'react';
 import './App.css';
+
+import { connect } from 'react-redux';
+import { fetchSmurfs, deleteSmurf, updateSmurf } from '../actions';
+
+import SmurfForm from './SmurfForm';
+import SmurfList from './SmurfList';
 /*
  to wire this component up you're going to need a few things.
  I'll let you do this part on your own. 
  Just remember, `how do I `connect` my components to redux?`
  `How do I ensure that my component links the state to props?`
  */
+
 class App extends Component {
+  state = {
+    editForm: false,
+  }
+
+  componentDidMount() {
```

Comment:
Awesome work fetching the smurfs in componentDidMount! Note that doing this here sort of defeats the purpose of redux, since you are still passing the smurfs down through props